tests/network/vmi_multus.go

```diff
@@ -198,32 +198,30 @@ var _ = SIGDescribe("[Serial]Multus", func() {
 	})
 
 	createVMIOnNode := func(interfaces []v1.Interface, networks []v1.Network) *v1.VirtualMachineInstance {
-		vmi := tests.NewRandomVMIWithEphemeralDiskAndUserdata(cd.ContainerDiskFor(cd.ContainerDiskAlpine), tests.BashHelloScript)
+		vmi := libvmi.NewAlpine()
 		vmi.Spec.Domain.Devices.Interfaces = interfaces
```

Comment:
Is `BashHelloScript` still used anywhere? Or it can be removed?

Reply:
I'm afraid it is still used by a storage test, which would be a bit harder to untangle as we do not have a `libvmi` factory for a PVC-based VM. I hope @brybacki can look into it.